src/sparseml/transformers/sparsification/trainer.py

```diff
@@ -140,8 +140,7 @@ def __init__(
         else:
             self.logger_manager = LoggerManager(log_python=False)
 
-        # remove arch_managers once recipe stages are supported
-        self.manager, self.arch_managers = self._setup_manager(kwargs)
+        self.manager = self._setup_manager(kwargs)
```

Comment:
should have checkpoint manager here as well still (you can think of all arch managers as being condensed into a single staged checkpoint recipe)